Sources/Quick/DSL/World+DSL.swift

```diff
@@ -134,4 +158,18 @@ extension World {
     internal func pending(description: String, closure: () -> ()) {
         print("Pending: \(description)")
     }
+
+    private var currentPhase: String {
+        if currentExampleMetadata != nil {
+            if beforesCurrentlyExecuting {
+                return "beforeEach"
+            } else if aftersCurrentlyExecuting {
+                return "afterEach"
+            }
+
+            return "it"
+        }
+
+        return ""
```

Comment:
This is only ever called from within `guard` statements that check for `currentExampleMetadata`; it's not meant to be used in any other situation. Furthermore, it doesn't use that value. So why not have it return `"it"` by default, as in:

```swift
private var currentPhase: String {
    if beforesCurrentlyExecuting {
        return "beforeEach"
    } else if aftersCurrentlyExecuting {
        return "afterEach"
    }
    return "it"
}
```

And lose the checks on `currentExampleMetadata`? You could include a comment to indicate when it should be used, or add a `precondition` that checks that `currentExampleMetadata != nil`.